### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by amuino** [kslazarev/numbers_and_words#118]
We have been using this gem for quite some time and are interested in getting some of the newer changes (specially #116).

Are there plans to make a new release to rubygems in the short term?

**Comment by miks** [comment]
It's up to @kslazarev. I have no permissions to push this gem to rubygems.

**Comment by amuino** [comment]
In the mean time I am releasing master in a forked gem: https://rubygems.org/gems/numbers_and_words_amuino

**Comment by jlduran** [comment]
Or you can include the gem from GitHub in your `Gemfile`:

```ruby
gem 'numbers_and_words', git: 'git://github.com/kslazarev/numbers_and_words'
```

**Comment by amuino** [comment]
In our case, we need to add the dependency to the `gemspec`, which does not allow references to git (as far as aI know)

**Comment by dblock** [comment]
Please add your +1 to forking this, https://github.com/kslazarev/numbers_and_words/issues/125.

**Comment by dblock** [comment]
I've published 0.10.5. We now have me and @miks both with r/w access and gem ownership on Rubygems. If anyone also has r/w access to this repo that we don't know about, please speak up and I'll add you to Rubygems as well.
